include/zephyr/posix/stropts.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Abhinav Srivastava
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_POSIX_STROPTS_H_
+#define ZEPHYR_INCLUDE_POSIX_STROPTS_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct {
+	int maxlen;
+	int len;
+	char *buf;
+} strbuf;
```

Comment:
I think this one line 
```suggestion
};
```

Reply:
Done